### ISSUE TITLE
Feature Request: Sparsemax activation

### ISSUE THREAD

**Issue by old-school-kid** [keras-team/keras#20554]
Hi Team. I have not committed to this repository since long and so do not know the guidelines for adding new activation function or loss function and so on. But Sparsemax is quite essential from MoE and sparse networks point of view. As a matter of the [original paper ](https://arxiv.org/abs/1602.02068) also has ~900 citations. Please let me know if this aligns with the team's feature addition policy post which I am more than happy to create a PR for this.

**Comment by fchollet** [comment]
Sure, we can review a PR for this, that's definitely enough citations. Thanks!

**Comment by old-school-kid** [comment]
Raised a (PR)[https://github.com/keras-team/keras/pull/20558]

**Comment by old-school-kid** [comment]
Closing this since the PR got merged.
